src/GraphQL/Extensions/ObjectExtensions.cs

```diff
@@ -184,20 +223,15 @@ public struct MemberFieldInfo
                 if (index == -1)
                 {
                     if (ctorParam.IsOptional)
-                        ctorFields[i] = new() { ParameterInfo = ctorParam };
+                        ctorFields[i] = new(null, ctorParam, null);
```

Comment:
Usually, I add argument names when passing constants. It's hard to understand who these `null` arguments are without checking the method/constructor signature. And absolutely impossible when reading the code in web browser. Compare with this:
```c#
new(key: null, ctorParam, graphType: null)
```
IMO much cleaner and easier to read.

Reply:
Good idea.